Examples/BabelES6/webpack.config.js

```diff
@@ -46,16 +47,17 @@ if (process.env.HOT) {
   config.entry['index.ios'].unshift('webpack-dev-server/client?http://localhost:8082');
   config.output.publicPath = 'http://localhost:8082/';
   config.plugins.unshift(new webpack.HotModuleReplacementPlugin());
-  config.module.loaders[0].query.plugins.push('react-transform');
-  config.module.loaders[0].query.extra = {
-    'react-transform': {
+
+  // Note: enabling React Transform and React Transform HMR:
+  config.module.loaders[0].query.plugins.push([
+    'react-transform', {
       transforms: [{
-        transform: 'react-transform-hmr',
-        imports: ['react-native'],
-        locals: ['module']
+        transform : 'react-transform-hmr',
+        imports   : ['react-native'],
+        locals    : ['module']
```

Comment:
that's a very uncommon style. can you please revert to `propertyName: [...]` without any whitespace? thanks!